app.js

```diff
@@ -58,6 +62,25 @@ const init = async () => {
     res.json(version);
   });
 
+  // Our specification download service can't use GraphQL because GraphQL specification requires the content-type
+  // that it returns be json, and we want to be able to return other content types, such as tab-separated-values,
+  // so that the user is automatically prompted to save the file from their browser.
+  const apiRoot = urlJoin(CLINICAL_SERVICE_ROOT, SUBMISSION_TEMPLATE_PATH);
+  app.get('/downloads', async (req,res) => {
+    const name = req.query.template;
+    if (name == undefined) {
+      res.json({error: "Invalid query: use parameter template to specify the desired template file."});
+    } else {
+      const data = fetch(urlJoin(apiRoot, name), {synchronous: true}).then( (r) => {
```

Comment:
I couldn't find this `synchronous` option in the api docs, could you send me a reference for what it does and why we are using it?

Reply:
It runs the fetch synchronously, and we're using it because I forgot to switch back to async once I got things working. ;-) Will fix.